Doc/library/interpreters.rst

```diff
@@ -0,0 +1,173 @@
+:mod:`interpreters` --- High-level Subinterpreters Module
+==========================================================
+
+.. module:: interpreters
+   :synopsis: High-level SubInterpreters Module.
+
+**Source code:** :source:`Lib/interpreters.py`
+
+--------------
+
+This module provides tools for working with sub-interpreters, such as creating them,
+running code in them, or sending data between them.  It is a wrapper around the low-
+level :mod:`_interpreters` module.
+
+.. versionchanged:: added in 3.9
+
+Interpreter Objects
+-------------------
+
+The Interpreter object represents a single interpreter.
+.. class:: Interpreter(id)
+
+    The class implementing a subinterpreter object.
+
+    .. method:: is_running()
+
+       Return whether or not the identified interpreter is running.
+
+    .. method:: destroy()
+
+       Destroy the identified interpreter. Attempting to destroy the current
+       interpreter results in a RuntimeError. So does an unrecognized ID.
```

Comment:
I think current is not a good wording here, what do you mean by current. From [PEP-554](https://www.python.org/dev/peps/pep-0554/#interpreters-module-api) the idea was to not be able destroy a running interpreter.

Reply:
The last sentence is also not clear, what ID, how is it unrecognised?